crypto-ffi/bindings/gradle/libs.versions.toml

```diff
@@ -11,7 +11,7 @@ android-junit = "1.1.5"
 android-logback = "2.0.0"
 android-tools = "8.0.0"
 sdk-compile = "33"
-sdk-min = "26"
+sdk-min = "33"
```

Comment:
Why this change? If this is unrelated to the other changes in this commit, I'd prefer if this was moved into its own, separate commit, together with a message describing the "why".

Reply:
When building the generated android bindings the compiler said it was using things SDK level 33. But you're I'll alright try to reproduce this on main and make separate PR for it.

Reply:
Ok, we should definitely not raise the `sdk-min` to 33.

This comes from https://github.com/mozilla/uniffi-rs/issues/2226

I'll fix this in separate PR, the cleaner implementation is supposed to fallback to something else on older phones. I'll investigate a bit more if this is just linter issue or coming from the fact that we share code between android and java.

Reply:
Ok, reverted this and issue was fixed in https://github.com/wireapp/core-crypto/pull/715